### ISSUE TITLE
Color coding of tasks according to their status possible?

### ISSUE THREAD

**Issue by cscheidsteff** [yanivsegev/VSTS-DropPlan#121]
We work in project teams of usually between 5-10 members and often have to manage a larger number of workitems per sprint. For a better overview, it would be very helpful for us to recognize the status of a task ("Todo", "Doing", etc.) or also its severity in the drop plan. For example, via a corresponding color or graphic marker.

**Comment by sgtwilko** [comment]
We do have the indicators at the bottom of the task which pick up the status colour.

So the dots at the bottom left show the current and related tasks.
The related tasks are alpha'd a bit, so that the status of the current task is more obvious.

If you are using time logging you can see the in progress tasks as they fill up (see the readme for screenshots of this in action)

The tasks turn green once complete, so really it's just the "in progress" we'd need to highlight.

We could show the status if there's no time tracking in the bottom right of the ticket.  Although this could be an issue with the changes made for over 10 child task per parent.

WRT priority, we could highlight P1/P2 with an icon in the top right, like the warning icon.

**Comment by cscheidsteff** [comment]
Thank you for your feedback and suggestions. We have discussed this and have a few comments and questions below, also to see if we have understood everything correctly:  

> So the dots at the bottom left show the current and related tasks...

the relatedTaskBox is actually only displayed if there are also related tasks, right? As we understand it, in this box the dot on the right represents the Current Task. However, this is apparently always displayed in dark gray. Would the color of this dot indicate the status of the current task in the future and would the relatedTaskBox  then always be visible?

> If you are using time logging you can see the in progress tasks as they fill up...

We know about this functionality, but unfortunately we are not using time logging yet and it's not really a progress indicator that we need, but simply the information whether a task has been started

> The tasks turn green once complete, so really it's just the "in progress" we'd need to highlight

an additional color coding for "In Progress" would be an obvious solution and the first thing we thought of. However, this would then also compete with the indication for "Overdue". A task can be both at the same time. Could perhaps coloring the ::Before element be an alternative instead?

> WRT priority, we could highlight P1/P2 with an icon in the top right, like the warning icon.

this would be great!

> actually I've had a thought about modifying the highlight to be a drop down containing None, Area path, Parent. We could add a Status to this

We didn't really understand how this is meant. Could you explain this in more detail?

**Comment by sgtwilko** [comment]
I wrote a long response, but binned it as the saliant part is this:

I've tried changing the existing "Highlight area path" feature to allow us to highlight other things.

Here I've changed it to highlight on Status:
![image](https://github.com/yanivsegev/VSTS-DropPlan/assets/658876/abefa0b3-12e2-4d5b-81f9-6fdb450b9ff6)

The QA task shows as blue as it's "Active", and the Testing task shows as grey as it's "New", the plan picks up your configured colours.
The overdue tasks still show with the overdue colour, but the others have changed background to indicate "New" and "Active".

I'll continue to tweak this and look at a priority indicator.
